<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VRCWSensorTrackSysDimClass</class>
 <widget class="QWidget" name="VRCWSensorTrackSysDimClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="VRCWSensorTrackSysDimClassVLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignLeft|Qt::AlignTop">
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QVBoxLayout" name="mainWidgetVLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QWidget" name="trackSysSensDescLabelWidget" native="true">
        <layout class="QHBoxLayout" name="trackSysSensDescLabelHLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="sensTrackSysLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>sensTrackSys</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="descLabel">
           <property name="text">
            <string>description</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QWidget" name="sensTrackSysWidget" native="true">
        <layout class="QGridLayout" name="handGLayout">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>20</number>
         </property>
         <property name="verticalSpacing">
          <number>8</number>
         </property>
         <item row="0" column="0" alignment="Qt::AlignLeft">
          <widget class="QLabel" name="OffsetLabel">
           <property name="text">
            <string>Offset [mm]</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="OffsetHLayout">
           <item>
            <widget class="QLabel" name="OffsetXLabel">
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignLeft">
            <widget class="QSpinBox" name="OffsetXSpinBox">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="minimum">
              <number>-9999</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="OffsetHSpacer1">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="OffsetYLabel">
             <property name="text">
              <string>y</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignLeft">
            <widget class="QSpinBox" name="OffsetYSpinBox">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="minimum">
              <number>-9999</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="OffsetHSpacer2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="OffsetZLabel">
             <property name="text">
              <string>z</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignLeft">
            <widget class="QSpinBox" name="OffsetZSpinBox">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="minimum">
              <number>-9999</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" alignment="Qt::AlignLeft">
          <widget class="QLabel" name="OrientationLabel">
           <property name="text">
            <string>Orientation [degree]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="OrientationHLayout">
           <item>
            <widget class="QLabel" name="OrientationHLabel">
             <property name="text">
              <string>h</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignLeft">
            <widget class="QSpinBox" name="OrientationHSpinBox">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="minimum">
              <number>-180</number>
             </property>
             <property name="maximum">
              <number>180</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="OrientationHSpacer1">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="OrientationPLabel">
             <property name="text">
              <string>p</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignLeft">
            <widget class="QSpinBox" name="OrientationPSpinBox">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="minimum">
              <number>-180</number>
             </property>
             <property name="maximum">
              <number>180</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="OrientationHSpacer2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="OrientationRLabel">
             <property name="text">
              <string>r</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignLeft">
            <widget class="QSpinBox" name="OrientationRSpinBox">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="minimum">
              <number>-180</number>
             </property>
             <property name="maximum">
              <number>180</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
